expression/builtin_string_vec_test.go

```diff
@@ -58,7 +58,12 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString, types.ETString, types.ETString}},
 	},
 	ast.ConcatWS: {},
-	ast.Convert:  {},
+	ast.Convert: {
```

Comment:
Please add more cases to cover more encoding methods.

Reply:
ok